format

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Script uses clang-format to format all of the source files in a
+# default format.
+#
+
+# The absolute path of the directory containing this script.
+DIR="$( cd "$( dirname "$0" )" && pwd)"
+# Define top level project directory
+PROJECT_DIR="${DIR}"
+
+METHOD="GIT"
+if [ -n "$1" ]; then
+ METHOD=$1
+fi
+
+# Paths to format
+MONITOR=()
+MONITOR+=( "${PROJECT_DIR}/Source" )
+
+if [ "$METHOD" == "ALL" ]; then
+    # Format all Source files
+    for FILE in $(find ${MONITOR[*]} -type f \( -iname "*.mm" -o -iname "*.m" -o -iname "*.h" \))
+    do
+        clang-format -style=file -i "$FILE" &
```

Comment:
```suggestion
        clang-format -style=file -i "$FILE"
```

Reply:
The `&` is there on purpose. It runs the command in the background and does not wait for every file.

Reply:
ahh, got it!